harper-ls/src/backend.rs

```diff
@@ -206,6 +208,8 @@ impl Backend {
                 }
             } else if language_id == "markdown" {
                 Some(Box::new(Markdown))
+            } else if language_id == "typst" {
```

Comment:
Seems to be working for me! Also seems that it [mason-lspconfig.nvim](https://github.com/williamboman/mason-lspconfig.nvim/blob/43894adcf10bb1190c2184bd7c1750e8ea2b3dce/lua/mason-lspconfig/mappings/filetype.lua#L214) uses that as the language ID as well.

Reply:
Yep, looks right, I'm not sure exactly what the correct way to modify `mason-lspconfig.nvim` since the important files say they are autogenerated.

Reply:
Me neither but [this is how harper was added last time](https://github.com/williamboman/mason-lspconfig.nvim/pull/406/files). My guess is that it would also need to be added to [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig)

Reply:
I think so too, I've made a draft pr now in the lspconfig repo.